policyengine_us/variables/gov/states/mt/tax/income/exemptions/mt_exemptions_joint.py

```diff
@@ -0,0 +1,17 @@
+from policyengine_us.model_api import *
+
+
+class mt_exemptions_joint(Variable):
+    value_type = float
+    entity = Person
+    label = "Montana exemptions when married couple files jointly"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.MT
+
+    def formula(person, period, parameters):
+        # Allocate the exemptions to the head
+        head = person("is_tax_unit_head", period)
+        exemption_count = add(person.tax_unit, period, ["mt_exemptions_indiv"])
```

Comment:
This is amount not count based on the variable
```suggestion
        exemptions = add(person.tax_unit, period, ["mt_exemptions_indiv"])
```